cmd/register/files.go

```diff
@@ -19,7 +21,13 @@ If there are already registered entities with v1 version then the command will f
 ::
 
  bin/flytectl register file  _pb_output/* -d development  -p flytesnacks
+	
+Fast Registers will register all the fast serialized protobuf files including tasks, workflows and launchplans with default v1 version. 
+If there are already registered entities with v1 version then the command will fail immediately on the first such encounter.
```

Comment:
You have --continueOnError in the example command but you have mentioned fail on first encounter. Probably remove the continueOnError to be in sync